src/xdocs/google_style.xml

```diff
@@ -21,14 +21,14 @@
                 <p>
                     This coverage report was created for
                     <a href="http://google.github.io/styleguide/javaguide.html">
-                        Google Java Style</a>(<a
-                        href="styleguides/google-java-style-20180523/javaguide.html"
-                    >cached page</a>),
+                        Google Java Style</a>(
+                    <a href="styleguides/google-java-style-20180523/javaguide.html">
+                    cached page</a>),
                     version of 23 May 2018, current as of 07 May 2019
                 </p>
 
                 <p>
-                    <a href="http://checkstyle.sourceforge.net/reports/google-style/guava/">
+                    <a href="http://checkstyle.org/reports/google-style/guava/">
```

Comment:
This change is not related to this issue.

Reply:
"reports" are only located in sourceforge.io , please use sourceforge.io

Reply:
done